Jenkinsfile_nightly

```diff
@@ -30,9 +31,10 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 }
 
 withNightlyPipeline("nodejs", product, component) {
-    env.TEST_E2E_URL = params.URL_TO_TEST
+    env.TEST_CCD_E2E_URL = params.CCDUI_URL_TO_TEST
+    env.TEST_XUI_E2E_URL = params.XUI_URL_TO_TEST
     loadVaultSecrets(secrets)
-    enableFullFunctionalTest()
+    enableFullFunctionalTest(90)
```

Comment:
increasing nightly functional test timeout value to 90 min. Currently it is taking around 40 to 60 mins.